framework/test/unit/modules/interoperability/sidechain/commands/initialize_state_recovery.spec.ts

```diff
@@ -313,11 +335,11 @@ describe('Sidechain InitializeStateRecoveryCommand', () => {
 			);
 
 			await expect(stateRecoveryInitCommand.verify(commandVerifyContext)).rejects.toThrow(
-				'Sidechain is still active and obeys the liveness requirement.',
+				'Sidechain has status registered.',
 			);
 		});
 
-		it('should return error if the sidechain has ChainStatus.REGISTERED status', async () => {
+		it('should return error if the sidechain is active on the mainchain and does not violate the liveness requirement', async () => {
```

Comment:
Perhaps add one more test that in case i) sidechain is active and ii) it DOES violate the liveness requirement, it does not throw.